packages/block-library/src/latest-posts/edit.js

```diff
@@ -82,11 +82,11 @@ class LatestPostsEdit extends Component {
 					/>
 					{ displayPostContent &&
 					<RadioControl
-						label="Show:"
+						label= __( 'Show:' )
```

Comment:
This is a syntax error.

```suggestion
						label={ __( 'Show:' ) }
```

Reply:
how did I miss this :)

Reply:
Fixed :)